sdk/eventhubs/azure-messaging-eventhubs-checkpointstore-redis/src/main/java/com/azure/messaging/eventhubs/checkpointstore/redis/JedisRedisCheckpointStoreBuilder.java

```diff
@@ -0,0 +1,94 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Since this is package-private...  can we just delete it?

Reply:
As in, delete the comment?

Reply:
Delete the entire class